packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -214,10 +217,15 @@ export default class VListItem {
      * Apply the list style type
      * @param rootList the vList that receives the style
      * @param index the list item index
+     * @param numberDefinition the number definitions of BulletListType or NumberingListType
      */
-    applyListStyle(rootList: HTMLOListElement | HTMLUListElement, index: number) {
-        const style = getMetadata(rootList) ? getMetadata(rootList) : undefined;
-        if (style) {
+    applyListStyle(
+        rootList: HTMLOListElement | HTMLUListElement,
+        index: number,
+        numberDefinition: NumberDefinition
```

Comment:
I don't think it is a good idea to expose the numberDefinition as a parameter of a public API. That means every one can call this API with a different value of definition, which is not correct.